utils/regression-analysis/src/runtime.rs

```diff
@@ -0,0 +1,118 @@
+use crate::weights::BenchmarkWeight;
```

Comment:
can't we avoid building here runtime?
we can just ask node to benchmark everything with our maintain template and work around json outputs